source/jobs/developer.erb

```diff
@@ -6,43 +6,43 @@ header:
   subtitle: "Ruby on Rails Developer"
 
 introduction:
-  title: "Are you a Ruby on Rails developer?"
-  text_1: "We are looking for Ruby on Rails developers to join our London office."
-  text_2: "Recognised as being key members in the community, our developers are responsible for building scalable, sustainable systems from project inception right through to production."
-  text_3: "We need people who can:"
+  title: "Are you a Ruby on Rails Developer?"
+  text_1: "We are on the lookout for Ruby on Rails Developers to join us in London. As a core part of our team, our developers are responsible for building scalable, sustainable systems from project inception right through to production."
+  text_2: "With the aim of broadening the diversity across our company, we are actively committed to building a more inclusive team."
+  text_3: "Some of the key activities within this role include:"
   requirements:
-    - Develop and maintain complex Ruby on Rails applications through agile ways of working
+    - Developing and maintaining Ruby on Rails applications through agile ways of working
     - Make informed decisions quickly drawing from experience across a wide technical stack
+    - Turning designs and wireframes into functional applications
     - Involve your team in discussions on progress of stories
-    - Turn designs and wireframes into functional applications
+    - Identifying areas of risk and complexity during each project
+    - Collaborating with designers, user researchers, subject matter experts and other project stakeholders
 
 developers_image:
   image: "/jobs/images/developers-1.jpg"
   image_description: "Developers presenting code"
 
 description:
-  title: "Key competencies and skills"
+  title: "Some of the skills we're looking for"
+  intro: "These are the types of skills that we expect our developers to have, but if there's anything you can't do yet, we will help you learn:"
```

Comment:
Nitpick - I'm not sure if this is the right grammar. Maybe drop the first comma and put parentheses around everything that follows?

Reply:
Amended this in the commit too